### PR TITLE
refactor: update merged performance table colors

### PR DESCRIPTION
**Pull request by DanielPuthawala**
Fixed colors of table 2

### CODE REVIEW

File: analysis/performance_analysis/merged_performance_analysis.ipynb [hunk 1]
```diff
@@ -1,29 +1,5 @@
...
-    "<!-- vscode-jupyter-toc-config\n",
-    "\tnumbering=false\n",
-    "\tanchor=true\n",
-    "\tflat=false\n",
-    "\tminLevel=1\n",
-    "\tmaxLevel=6\n",
-    "\t/vscode-jupyter-toc-config -->\n",
-    "<!-- THIS CELL WILL BE REPLACED ON TOC UPDATE. DO NOT WRITE YOUR TEXT IN THIS CELL -->"
-   ]
-  },
```
Comment (korikuzma) [inline analysis/performance_analysis/merged_performance_analysis.ipynb:26]: Why did you delete the table of contents?

**Review by wesleygoar — DISMISSED** [review]

File: analysis/performance_analysis/merged_performance_analysis.ipynb (hunk 1 above)
Comment (wesleygoar) [reply analysis/performance_analysis/merged_performance_analysis.ipynb:26]: I am also wondering about that

Comment (DanielPuthawala) [reply analysis/performance_analysis/merged_performance_analysis.ipynb:26]: Oh did I?  Weird!

Comment (DanielPuthawala) [reply analysis/performance_analysis/merged_performance_analysis.ipynb:26]: Lemme re-insert it.

**Comment by DanielPuthawala** [comment]
Weird, it's not letting my push my commits.
"Can't push refs to remote. Try running "Pull" first to integrate your changes."

**Comment by DanielPuthawala** [comment]
@korikuzma does the process to make and push commits change once you've submitted a PR?

**Comment by korikuzma** [comment]
> @korikuzma does the process to make and push commits change once you've submitted a PR?

@DanielPuthawala no. Have you tried following the suggested prompts?

**Comment by DanielPuthawala** [comment]
Yes, I Pulled, and then when I try to push again I get the same error.

**Comment by korikuzma** [comment]
@DanielPuthawala can you comment with the exact commands you're running and the errors you're getting?

**Comment by DanielPuthawala** [comment]
Git: Push
> Can't push refs to remote. Try running "Pull" first to integrate your changes.
Git: Pull
>Git: From https://github.com/GenomicMedLab/variation-normalizer-manuscript
Git: Push
> Can't push refs to remote. Try running "Pull" first to integrate your changes.
I've tried restarting VScode, undoing the commit, pulling and then trying to re-commit and push the commit, witht he same error.  I've also switched over to main  and pulled, with no problem, then gone back to issue-92 and tried pushing again

**Comment by korikuzma** [comment]
@DanielPuthawala Ahh the VS Code errors aren't that helpful. Can you try `git pull --rebase`?

**Comment by DanielPuthawala** [comment]
Looks like that did it!
Ok, should have the toc back in place now.
